src/SearchBar.js

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+
+function SearchBar(props) {
```

Comment:
This is probably more of a future thing than something to implement now but you could destructure rather than using props. (this one is honestly just a thought)